### DISCUSSION TITLE
Again it showing roboshop.conf no such file or directiory sir

### DISCUSSION THREAD

**Discussion by Tirupathi1729** [learndevopsonline/learndevopsonline#1218] [Q&A]
![newfrontend](https://github.com/learndevopsonline/learndevopsonline/assets/80533527/351b79eb-fb2d-4396-ab05-92a10423d942)
![frontend intelliji](https://github.com/learndevopsonline/learndevopsonline/assets/80533527/aae7723c-4734-4eb0-b018-7e3a11277c61)
![conf intelliji](https://github.com/learndevopsonline/learndevopsonline/assets/80533527/812915f1-7578-41d0-a5f6-7ca807199c92)
![github new](https://github.com/learndevopsonline/learndevopsonline/assets/80533527/d3801fef-d147-4438-9444-1944532186f6)

how to fix it sir

**Comment by maheshbusireddy** [comment]
Hi Tirupathi
your path of frontend script and roboshop configuration file are in same path.

**Reply by Tirupathi1729** [reply]
yes

**Reply by maheshbusireddy** [reply]
you have to create one directory called files in your roboshop-shell repository. inside files directory create nginx-roboshop.conf and change path.
if you don t mind please re-watch video again.
